src/dotnet/APIView/APIViewWeb/Pages/Assemblies/RequestedReviews.cshtml.cs

```diff
@@ -30,7 +30,7 @@ public async Task<IActionResult> OnGetAsync()
             var fullResult = (await _manager.GetReviewsAsync(false, "All")).Where(r => r.RequestedReviewers != null).Where(r => r.RequestedReviewers.Contains(User.GetGitHubLogin()));
```

Comment:
I had missed this when I reviewed last time. A better approach is to fetch only required reviews from cosmos DB by adding a new query in cosmos repository class to fetch requested reviews. But this can be done as a separate PR to optimize the code.